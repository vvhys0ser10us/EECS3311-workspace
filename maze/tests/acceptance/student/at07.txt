new_game(easy)
solve
abort
new_game(easy)
solve
abort
new_game(easy)
solve
abort
new_game(easy)
solve
abort
new_game(easy)
solve
abort
new_game(medium)
solve
abort
new_game(medium)
solve
abort
new_game(medium)
solve
abort
new_game(medium)
solve
abort
new_game(hard)
solve
abort
new_game(hard)
solve
abort
new_game(hard)
solve
abort


