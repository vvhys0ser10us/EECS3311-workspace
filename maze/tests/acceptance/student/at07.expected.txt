  State: 0 -> ok
->new_game(easy)
  State: 1 -> ok
  +-*-+---+---+---+---+---+---+
  | X                         |
  +   +   +   +---+---+   +   +
  |       |   |       |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->solve
  State: 2 -> ok
  +-*-+---+---+---+---+---+---+
  | X   o   o                 |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->abort
  State: 3 -> ok
->new_game(easy)
  State: 4 -> ok
  +-*-+---+---+---+---+---+---+
  | X                 |       |
  +   +   +   +---+---+   +   +
  |   |               |       |
  +---+   +---+   +---+   +   +
  |       |   |           |   |
  +   +   +   +---+   +---+   +
  |                   |   |   |
  +   +   +   +   +   +   +   +
  |   |   |   |               |
  +---+---+---+   +---+   +   +
  |           |   |   |   |   |
  +   +   +---+---+   +   +   +
  |               |           |
  +---+---+---+---+---+---+- -+

  Game Number: 2
  Score: 0 / 2

->solve
  State: 5 -> ok
  +-*-+---+---+---+---+---+---+
  | X   o   o         |       |
  +   +   +   +---+---+   +   +
  |   |     o   o     |       |
  +---+   +---+   +---+   +   +
  |       |   | o   o     |   |
  +   +   +   +---+   +---+   +
  |                 o |   |   |
  +   +   +   +   +   +   +   +
  |   |   |   |     o   o   o |
  +---+---+---+   +---+   +   +
  |           |   |   |   | o |
  +   +   +---+---+   +   +   +
  |               |         o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 2
  Score: 0 / 2

->abort
  State: 6 -> ok
->new_game(easy)
  State: 7 -> ok
  +-*-+---+---+---+---+---+---+
  | X                         |
  +---+---+   +---+---+   +   +
  |               |   |   |   |
  +---+---+   +---+   +   +   +
  |           |           |   |
  +   +   +---+   +---+---+   +
  |                           |
  +---+---+   +---+---+---+   +
  |       |       |   |       |
  +   +   +   +   +   +   +   +
  |       |       |   |       |
  +   +---+   +---+   +---+   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 3
  Score: 0 / 3

->solve
  State: 8 -> ok
  +-*-+---+---+---+---+---+---+
  | X   o   o   o   o   o   o |
  +---+---+   +---+---+   +   +
  |               |   |   | o |
  +---+---+   +---+   +   +   +
  |           |           | o |
  +   +   +---+   +---+---+   +
  |                         o |
  +---+---+   +---+---+---+   +
  |       |       |   |     o |
  +   +   +   +   +   +   +   +
  |       |       |   |     o |
  +   +---+   +---+   +---+   +
  |       |                 o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 3
  Score: 0 / 3

->abort
  State: 9 -> ok
->new_game(easy)
  State: 10 -> ok
  +-*-+---+---+---+---+---+---+
  | X                         |
  +   +   +---+   +---+   +   +
  |       |           |   |   |
  +   +   +---+   +   +---+   +
  |       |       |   |       |
  +   +   +   +   +---+---+   +
  |   |               |       |
  +   +   +---+---+   +---+---+
  |   |                   |   |
  +   +   +   +---+   +   +   +
  |   |   |   |   |       |   |
  +   +   +   +   +---+---+   +
  |       |               |   |
  +---+---+---+---+---+---+- -+

  Game Number: 4
  Score: 0 / 4

->solve
  State: 11 -> Error! Maze is not solvable, you may abort with no penalty.
  Game Number: 4
  Score: 0 / 4

->abort
  State: 12 -> ok
->new_game(easy)
  State: 13 -> ok
  +-*-+---+---+---+---+---+---+
  | X     |   |               |
  +---+   +   +---+   +   +   +
  |   |       |       |       |
  +   +---+   +   +   +---+---+
  |   |   |               |   |
  +   +   +   +   +   +---+   +
  |           |   |   |       |
  +   +---+   +---+---+---+   +
  |       |           |       |
  +   +---+---+   +---+   +---+
  |   |       |           |   |
  +   +   +   +   +   +   +   +
  |                           |
  +---+---+---+---+---+---+- -+

  Game Number: 5
  Score: 0 / 4

->solve
  State: 14 -> ok
  +-*-+---+---+---+---+---+---+
  | X   o |   |               |
  +---+   +   +---+   +   +   +
  |   | o   o |       |       |
  +   +---+   +   +   +---+---+
  |   |   | o             |   |
  +   +   +   +   +   +---+   +
  |         o |   |   |       |
  +   +---+   +---+---+---+   +
  |       | o   o     |       |
  +   +---+---+   +---+   +---+
  |   |       | o   o   o |   |
  +   +   +   +   +   +   +   +
  |                     o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 5
  Score: 0 / 4

->abort
  State: 15 -> ok
->new_game(medium)
  State: 16 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X     |   |                           |   |
  +   +   +   +   +---+---+   +---+---+   +   +
  |   |           |   |       |   |   |       |
  +   +   +   +   +   +---+   +   +   +   +   +
  |       |       |       |       |       |   |
  +   +   +---+   +   +   +---+   +   +---+   +
  |   |   |           |   |   |   |   |       |
  +---+---+---+   +   +   +   +---+---+   +   +
  |   |   |       |       |               |   |
  +   +   +   +   +   +   +---+   +---+---+   +
  |               |   |       |               |
  +---+   +---+---+   +---+   +   +   +---+   +
  |               |       |               |   |
  +   +---+   +---+   +   +   +---+---+   +   +
  |   |   |       |       |           |       |
  +---+   +---+   +   +---+   +   +   +---+   +
  |   |       |           |   |               |
  +   +---+   +---+   +   +---+   +   +   +   +
  |       |   |       |   |   |           |   |
  +   +   +   +---+---+   +   +---+   +---+   +
  |               |                           |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 6
  Score: 0 / 6

->solve
  State: 17 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X   o |   |                           |   |
  +   +   +   +   +---+---+   +---+---+   +   +
  |   | o   o   o |   |       |   |   |       |
  +   +   +   +   +   +---+   +   +   +   +   +
  |       |     o |       |       |       |   |
  +   +   +---+   +   +   +---+   +   +---+   +
  |   |   |     o   o |   |   |   |   |       |
  +---+---+---+   +   +   +   +---+---+   +   +
  |   |   |       | o   o |               |   |
  +   +   +   +   +   +   +---+   +---+---+   +
  |               |   | o   o |               |
  +---+   +---+---+   +---+   +   +   +---+   +
  |               |       | o   o   o   o |   |
  +   +---+   +---+   +   +   +---+---+   +   +
  |   |   |       |       |           | o   o |
  +---+   +---+   +   +---+   +   +   +---+   +
  |   |       |           |   |             o |
  +   +---+   +---+   +   +---+   +   +   +   +
  |       |   |       |   |   |           | o |
  +   +   +   +---+---+   +   +---+   +---+   +
  |               |                         o |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 6
  Score: 0 / 6

->abort
  State: 18 -> ok
->new_game(medium)
  State: 19 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X         |                               |
  +   +---+   +---+---+   +   +   +---+   +   +
  |   |       |   |           |       |   |   |
  +   +   +   +   +   +---+---+---+---+---+   +
  |   |       |                   |       |   |
  +   +   +---+---+---+   +   +   +   +   +   +
  |           |               |               |
  +   +   +   +   +---+   +   +---+   +   +---+
  |   |       |   |   |   |   |           |   |
  +   +   +   +   +   +   +   +---+   +---+   +
  |       |                   |               |
  +---+   +---+   +---+   +---+   +   +---+   +
  |   |   |   |           |   |   |       |   |
  +   +   +   +   +---+   +   +   +---+   +---+
  |       |       |               |   |       |
  +---+---+---+   +---+---+---+---+   +   +---+
  |   |   |   |           |                   |
  +   +   +   +---+   +   +   +   +---+---+   +
  |       |                   |   |   |   |   |
  +---+   +   +---+---+---+   +   +   +   +   +
  |                       |   |   |       |   |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 7
  Score: 0 / 8

->solve
  State: 20 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X         |                               |
  +   +---+   +---+---+   +   +   +---+   +   +
  | o |       |   |           |       |   |   |
  +   +   +   +   +   +---+---+---+---+---+   +
  | o |       |                   |       |   |
  +   +   +---+---+---+   +   +   +   +   +   +
  | o   o   o |               |               |
  +   +   +   +   +---+   +   +---+   +   +---+
  |   |     o |   |   |   |   |           |   |
  +   +   +   +   +   +   +   +---+   +---+   +
  |       | o   o             |               |
  +---+   +---+   +---+   +---+   +   +---+   +
  |   |   |   | o         |   |   |       |   |
  +   +   +   +   +---+   +   +   +---+   +---+
  |       |     o |               |   |       |
  +---+---+---+   +---+---+---+---+   +   +---+
  |   |   |   | o   o   o | o   o   o   o   o |
  +   +   +   +---+   +   +   +   +---+---+   +
  |       |             o   o |   |   |   | o |
  +---+   +   +---+---+---+   +   +   +   +   +
  |                       |   |   |       | o |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 7
  Score: 0 / 8

->abort
  State: 21 -> ok
->new_game(medium)
  State: 22 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X     |                                   |
  +   +---+   +   +---+   +   +---+   +   +   +
  |                   |       |   |           |
  +---+   +---+   +---+   +---+   +   +---+   +
  |   |   |   |       |   |           |       |
  +   +---+   +   +---+   +---+---+---+   +   +
  |                           |           |   |
  +   +---+---+---+   +---+---+   +   +---+   +
  |   |       |       |   |   |   |           |
  +---+   +   +   +   +   +   +---+   +---+   +
  |           |   |   |               |       |
  +   +---+   +   +   +---+   +---+---+---+---+
  |       |   |   |       |       |   |       |
  +   +   +---+---+---+   +   +---+   +   +   +
  |               |           |               |
  +   +   +   +---+   +   +---+   +   +---+---+
  |   |   |       |   |       |       |       |
  +   +---+   +   +---+   +   +   +   +---+   +
  |       |   |       |   |   |   |   |       |
  +   +   +   +---+   +   +   +   +   +   +   +
  |   |           |       |               |   |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 8
  Score: 0 / 10

->solve
  State: 23 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X     |                                   |
  +   +---+   +   +---+   +   +---+   +   +   +
  | o   o   o   o     |       |   |           |
  +---+   +---+   +---+   +---+   +   +---+   +
  |   |   |   | o     |   |           |       |
  +   +---+   +   +---+   +---+---+---+   +   +
  |             o   o         |           |   |
  +   +---+---+---+   +---+---+   +   +---+   +
  |   |       |     o |   |   |   |           |
  +---+   +   +   +   +   +   +---+   +---+   +
  |           |   | o |               |       |
  +   +---+   +   +   +---+   +---+---+---+---+
  |       |   |   | o   o |       |   |       |
  +   +   +---+---+---+   +   +---+   +   +   +
  |               |     o     |               |
  +   +   +   +---+   +   +---+   +   +---+---+
  |   |   |       |   | o   o |       |       |
  +   +---+   +   +---+   +   +   +   +---+   +
  |       |   |       |   | o |   |   | o   o |
  +   +   +   +---+   +   +   +   +   +   +   +
  |   |           |       | o   o   o   o | o |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 8
  Score: 0 / 10

->abort
  State: 24 -> ok
->new_game(medium)
  State: 25 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X                 |           |       |   |
  +   +---+   +---+   +   +---+   +   +   +   +
  |   |       |   |           |       |   |   |
  +---+---+   +   +   +   +---+   +   +---+   +
  |           |   |       |   |               |
  +---+---+   +   +---+   +   +---+---+   +---+
  |           |               |   |   |       |
  +   +---+   +   +---+---+   +   +   +   +   +
  |   |       |   |           |       |   |   |
  +   +---+---+---+   +---+---+---+   +   +   +
  |   |   |               |   |               |
  +   +   +   +---+   +   +   +   +   +   +   +
  |                   |           |           |
  +---+---+   +   +   +   +---+---+   +   +---+
  |           |   |   |       |       |   |   |
  +   +---+   +   +   +   +   +---+---+---+   +
  |               |       |   |   |       |   |
  +---+   +   +---+---+---+   +   +---+   +   +
  |           |       |                       |
  +   +---+   +---+   +---+   +   +   +   +   +
  |                       |       |   |       |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 9
  Score: 0 / 12

->solve
  State: 26 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+
  | X   o   o   o   o |           |       |   |
  +   +---+   +---+   +   +---+   +   +   +   +
  |   |       |   | o   o     |       |   |   |
  +---+---+   +   +   +   +---+   +   +---+   +
  |           |   |     o |   |               |
  +---+---+   +   +---+   +   +---+---+   +---+
  |           |         o   o |   |   |       |
  +   +---+   +   +---+---+   +   +   +   +   +
  |   |       |   | o   o   o |       |   |   |
  +   +---+---+---+   +---+---+---+   +   +   +
  |   |   |         o   o |   |               |
  +   +   +   +---+   +   +   +   +   +   +   +
  |                   | o         |           |
  +---+---+   +   +   +   +---+---+   +   +---+
  |           |   |   | o   o |       |   |   |
  +   +---+   +   +   +   +   +---+---+---+   +
  |               |       | o |   |       |   |
  +---+   +   +---+---+---+   +   +---+   +   +
  |           |       |     o   o   o   o   o |
  +   +---+   +---+   +---+   +   +   +   +   +
  |                       |       |   |     o |
  +---+---+---+---+---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 9
  Score: 0 / 12

->abort
  State: 27 -> ok
->new_game(hard)
  State: 28 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+---+---+---+---+
  | X                 |                                       |
  +---+   +   +   +   +---+---+---+---+   +---+   +---+   +   +
  |       |   |       |   |       |   |       |           |   |
  +   +---+   +   +---+   +   +   +   +   +---+---+---+   +---+
  |   |   |               |                   |       |       |
  +   +   +   +   +   +   +---+   +   +---+---+   +   +   +   +
  |   |   |   |       |   |           |           |       |   |
  +---+   +   +   +---+   +   +---+   +---+   +---+   +---+   +
  |   |       |   |               |       |   |       |   |   |
  +   +---+---+   +   +   +   +   +---+   +   +---+---+   +   +
  |           |   |   |   |                   |   |           |
  +---+---+   +   +   +   +   +---+---+---+---+   +---+---+   +
  |       |       |                       |   |       |   |   |
  +   +   +   +---+   +   +---+---+   +   +   +   +   +   +   +
  |   |       |   |               |                           |
  +---+   +---+   +   +   +---+---+---+---+---+---+---+   +   +
  |       |   |       |           |       |       |       |   |
  +   +---+   +   +---+---+   +---+   +   +   +---+   +---+   +
  |   |           |           |       |   |       |           |
  +   +---+---+---+   +   +   +---+   +   +   +   +---+   +   +
  |   |       |       |       |   |               |           |
  +   +   +   +   +   +   +---+   +   +   +---+---+---+   +---+
  |       |                   |           |   |               |
  +   +   +---+---+---+   +---+   +---+   +   +   +   +---+   +
  |           |       |       |       |           |   |   |   |
  +   +   +   +   +   +---+---+   +   +---+   +   +   +   +   +
  |   |   |       |       |       |               |           |
  +   +---+---+   +   +   +   +   +---+   +   +   +---+   +   +
  |       |       |   |   |           |       |           |   |
  +---+---+---+---+---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 10
  Score: 0 / 15

->solve
  State: 29 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+---+---+---+---+
  | X   o   o   o     |                                       |
  +---+   +   +   +   +---+---+---+---+   +---+   +---+   +   +
  |       |   | o     |   |       |   |       |           |   |
  +   +---+   +   +---+   +   +   +   +   +---+---+---+   +---+
  |   |   |     o   o   o |                   |       |       |
  +   +   +   +   +   +   +---+   +   +---+---+   +   +   +   +
  |   |   |   |       | o |           |           |       |   |
  +---+   +   +   +---+   +   +---+   +---+   +---+   +---+   +
  |   |       |   |     o   o     |       |   |       |   |   |
  +   +---+---+   +   +   +   +   +---+   +   +---+---+   +   +
  |           |   |   |   | o                 |   |           |
  +---+---+   +   +   +   +   +---+---+---+---+   +---+---+   +
  |       |       |         o   o   o   o |   |       |   |   |
  +   +   +   +---+   +   +---+---+   +   +   +   +   +   +   +
  |   |       |   |               |     o   o   o   o   o   o |
  +---+   +---+   +   +   +---+---+---+---+---+---+---+   +   +
  |       |   |       |           |       |       |       | o |
  +   +---+   +   +---+---+   +---+   +   +   +---+   +---+   +
  |   |           |           |       |   |       |     o   o |
  +   +---+---+---+   +   +   +---+   +   +   +   +---+   +   +
  |   |       |       |       |   |               |     o     |
  +   +   +   +   +   +   +---+   +   +   +---+---+---+   +---+
  |       |                   |           |   |         o   o |
  +   +   +---+---+---+   +---+   +---+   +   +   +   +---+   +
  |           |       |       |       |           |   |   | o |
  +   +   +   +   +   +---+---+   +   +---+   +   +   +   +   +
  |   |   |       |       |       |               |         o |
  +   +---+---+   +   +   +   +   +---+   +   +   +---+   +   +
  |       |       |   |   |           |       |           | o |
  +---+---+---+---+---+---+---+---+---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 10
  Score: 0 / 15

->abort
  State: 30 -> ok
->new_game(hard)
  State: 31 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+---+---+---+---+
  | X             |                               |           |
  +---+   +---+   +---+---+---+---+   +   +   +   +   +   +---+
  |   |       |               |       |   |               |   |
  +   +---+   +   +   +---+   +---+   +---+---+   +---+---+   +
  |       |   |       |       |       |           |   |   |   |
  +   +   +   +---+---+   +   +   +---+   +   +   +   +   +   +
  |       |   |   |       |   |       |       |               |
  +   +---+   +   +   +   +---+---+   +   +---+   +   +   +---+
  |   |           |   |   |                       |           |
  +   +   +   +   +   +---+---+---+---+   +---+   +   +---+   +
  |   |   |               |   |   |       |   |   |   |       |
  +---+   +   +---+---+   +   +   +---+---+   +---+---+   +   +
  |           |   |       |           |           |           |
  +---+   +---+   +   +   +---+   +   +   +   +---+   +   +   +
  |           |           |       |           |   |       |   |
  +   +   +---+   +   +   +   +   +---+   +---+   +   +   +---+
  |   |               |   |       |   |   |           |   |   |
  +---+---+---+---+   +---+   +   +   +   +---+---+---+   +   +
  |           |                                       |       |
  +   +---+---+   +---+---+---+---+   +---+---+---+   +   +---+
  |                   |           |   |       |   |       |   |
  +   +---+---+---+---+   +   +   +   +   +   +   +---+---+   +
  |               |   |       |       |   |               |   |
  +   +---+   +   +   +   +---+---+   +---+---+---+   +   +   +
  |               |       |           |   |   |           |   |
  +   +---+   +---+   +---+---+   +   +   +   +---+   +---+   +
  |           |       |   |   |           |       |           |
  +   +   +   +---+   +   +   +---+   +---+   +   +---+   +---+
  |   |                                       |               |
  +---+---+---+---+---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 11
  Score: 0 / 18

->solve
  State: 32 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+---+---+---+---+
  | X   o         |                               |           |
  +---+   +---+   +---+---+---+---+   +   +   +   +   +   +---+
  |   | o   o |               |       |   |               |   |
  +   +---+   +   +   +---+   +---+   +---+---+   +---+---+   +
  |       | o |       |       |       |           |   |   |   |
  +   +   +   +---+---+   +   +   +---+   +   +   +   +   +   +
  |       | o |   |       |   |       |       |               |
  +   +---+   +   +   +   +---+---+   +   +---+   +   +   +---+
  |   | o   o     |   |   |                       |           |
  +   +   +   +   +   +---+---+---+---+   +---+   +   +---+   +
  |   | o |               |   |   |       |   |   |   |       |
  +---+   +   +---+---+   +   +   +---+---+   +---+---+   +   +
  |     o     |   |       |           |           |           |
  +---+   +---+   +   +   +---+   +   +   +   +---+   +   +   +
  |     o     |           |       |           |   |       |   |
  +   +   +---+   +   +   +   +   +---+   +---+   +   +   +---+
  |   | o   o   o   o |   |       |   |   |           |   |   |
  +---+---+---+---+   +---+   +   +   +   +---+---+---+   +   +
  |           |     o   o   o   o   o                 |       |
  +   +---+---+   +---+---+---+---+   +---+---+---+   +   +---+
  |                   |           | o |       |   |       |   |
  +   +---+---+---+---+   +   +   +   +   +   +   +---+---+   +
  |               |   |       |     o |   |               |   |
  +   +---+   +   +   +   +---+---+   +---+---+---+   +   +   +
  |               |       |         o |   |   |           |   |
  +   +---+   +---+   +---+---+   +   +   +   +---+   +---+   +
  |           |       |   |   |     o     | o   o |           |
  +   +   +   +---+   +   +   +---+   +---+   +   +---+   +---+
  |   |                             o   o   o | o   o   o   o |
  +---+---+---+---+---+---+---+---+---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 11
  Score: 0 / 18

->abort
  State: 33 -> ok
->new_game(hard)
  State: 34 -> ok
  +-*-+---+---+---+---+---+---+---+---+---+---+---+---+---+---+
  | X     |   |   |       |                                   |
  +---+   +   +   +   +   +   +---+---+---+   +   +   +   +---+
  |       |           |           |   |       |       |       |
  +   +   +   +   +---+   +---+---+   +   +---+   +---+   +   +
  |           |                   |   |   |   |   |           |
  +   +   +   +---+---+   +   +   +   +   +   +---+   +   +   +
  |           |           |   |           |       |           |
  +---+   +   +---+---+---+   +   +   +---+   +   +---+---+---+
  |       |           |   |   |   |   |       |   |           |
  +   +   +---+---+   +   +---+---+---+   +   +   +   +   +---+
  |           |           |           |       |           |   |
  +   +---+---+---+   +---+---+---+   +   +---+   +   +   +   +
  |       |   |       |                       |               |
  +---+---+   +   +---+   +   +---+   +   +   +   +---+---+   +
  |   |   |           |   |       |   |       |   |   |       |
  +   +   +   +   +   +---+   +   +---+   +   +---+   +---+---+
  |           |   |           |       |               |   |   |
  +---+   +---+   +   +---+   +   +   +   +---+---+   +   +   +
  |       |           |       |       |       |   |   |       |
  +   +   +---+   +   +   +   +---+   +---+---+   +---+---+   +
  |   |           |           |               |   |   |       |
  +   +---+---+---+   +   +   +---+   +   +   +   +   +---+   +
  |   |               |   |       |           |           |   |
  +   +   +   +   +---+---+---+   +   +   +---+---+---+   +   +
  |           |   |   |   |               |   |               |
  +   +   +---+   +   +   +   +---+   +---+   +   +   +   +   +
  |   |       |   |               |   |   |   |               |
  +   +   +---+   +---+   +   +   +---+   +   +   +   +   +---+
  |   |   |       |           |           |       |   |       |
  +---+---+---+---+---+---+---+---+---+---+---+---+---+---+- -+

  Game Number: 12
  Score: 0 / 21

->solve
  State: 35 -> Error! Maze is not solvable, you may abort with no penalty.
  Game Number: 12
  Score: 0 / 21

->abort
  State: 36 -> ok
